src/PowerPing/Ping.cs

```diff
@@ -397,9 +399,14 @@ private PingResults SendICMP(PingAttributes attrs)
                         Display.RequestPacket(packet, Display.UseInputtedAddress | Display.UseResolvedAddress ? attrs.Host : attrs.Address, index);
                     }
 
+                    // If there were extra responses from a prior request, ignore them
+                    while (sock.Available != 0) {
```

Comment:
Checking the sockets availability every send cycle adds a small performance overhead, I'm wondering if there is a better way to check and flush the contents of the socket rather than this.